deploy/03_Deploy_RandomNumberConsumer.js

```diff
@@ -1,37 +1,41 @@
-const { getNamedAccounts, deployments, network } = require("hardhat")
+const { network } = require("hardhat")
 const {
   networkConfig,
   developmentChains,
   VERIFICATION_BLOCK_CONFIRMATIONS,
 } = require("../helper-hardhat-config")
-const { autoFundCheck, verify } = require("../helper-functions")
+const { verify } = require("../helper-functions")
 
 module.exports = async ({ getNamedAccounts, deployments }) => {
   const { deploy, get, log } = deployments
   const { deployer } = await getNamedAccounts()
   const chainId = network.config.chainId
   let linkTokenAddress
   let vrfCoordinatorAddress
-  let additionalMessage = ""
+  let subscriptionId
 
   if (chainId == 31337) {
     linkToken = await get("LinkToken")
-    VRFCoordinatorMock = await get("VRFCoordinatorMock")
+    VRFCoordinatorV2Mock = await ethers.getContract("VRFCoordinatorV2Mock")
+
+    vrfCoordinatorAddress = VRFCoordinatorV2Mock.address
     linkTokenAddress = linkToken.address
-    vrfCoordinatorAddress = VRFCoordinatorMock.address
-    additionalMessage = " --linkaddress " + linkTokenAddress
+
+    subscriptionId = await VRFCoordinatorV2Mock.callStatic.createSubscription()
```

Comment:
I think we don't want to `callStatic` here right? I think we want to call the actual function?  We can then just get the subscriptionId from the event log.